### ISSUE TITLE
[Docs] Setting up 2DC Replication With Backup Restore

### ISSUE THREAD

**Issue by rahuldesirazu** [yugabyte/yugabyte-db#6870]
Convert the following instructions into docs
-------------
These instructions detail setting up/catching up 2dc replication when unreplicated data has already been flushed to SST files and cannot be read using normal WAL based replication. As a prerequisite make sure to delete any existing replication stream if one already exists.

1) First, we need to create a checkpoint on the producer side for all the tables we want to replicate:

`yb-admin -master_addresses <producer_master_addresses> bootstrap_cdc_producer <comma_separated_producer_table_ids>`

This should return a list of bootstrap_ids, one per table id.

2) Then, backup the tables on the producer universe and restore to the consumer universe.

3) Finally, set up the replication stream, using the bootstrap_ids generated in step 1:

`yb-admin -master_addresses <consumer_master_addresses> setup_universe_replication <producer_universe_uuid> <producer_masters> <comma_separated_producer_table_ids> <bootstrap_ids>`

**Comment by lizayugabyte** [comment]
@rahuldesirazu Is this ticket still relevant? Because there's been a major overhaul of the 2DC documentation, which is now called Async Replication and published here: https://docs.yugabyte.com/latest/deploy/multi-dc/async-replication/

**Comment by rahuldesirazu** [comment]
Yes, these docs are still relevant, since they detail how to setup replication when the producer already has a lot of data. And I don't that's currently covered in our docs.

**Comment by CharlotteRose** [comment]
@rahuldesirazu  @lizayugabyte  - We will open a separate github for the issue if requested, but load balancing will need to be turned off on the destination cluster, or the restore can fail. 
see ZD ticket [1442](https://yugabyte.zendesk.com/agent/tickets/1442)

cc @tvesely @dotyjim-work

**Comment by polarweasel** [comment]
> We will open a separate github for the issue if requested, but load balancing will need to be turned off on the destination cluster, or the restore can fail. see ZD ticket [1442](https://yugabyte.zendesk.com/agent/tickets/1442)

@bmatican can you add a comment here on what we need to update?

**Comment by nspiegelberg** [comment]
User requested these docs in the Open Source slack: https://yugabyte-db.slack.com/archives/CG0KQF0GG/p1638813780465500

**Comment by nspiegelberg** [comment]
> > We will open a separate github for the issue if requested, but load balancing will need to be turned off on the destination cluster, or the restore can fail. see ZD ticket [1442](https://yugabyte.zendesk.com/agent/tickets/1442)
> 
> @bmatican can you add a comment here on what we need to update?

@polarweasel : The user needs to disable the `enable_load_balancing` GFLAG on the Producer yb-master process.

**Comment by Arjun-yb** [comment]
PR: https://github.com/yugabyte/yugabyte-db/pull/11625/files
Merged to master
